### DISCUSSION TITLE
How to display more generated labels on YAxis

### DISCUSSION THREAD

**Discussion by PeterKisTelus** [Live-Charts/LiveCharts2#647] [Q&A]
Hi,

if there are no labels set to YAxis, then it seems that the YAxis generate labels on its own:
![image](https://user-images.githubusercontent.com/113122190/192718755-6cd6af85-3880-4db2-8ff8-31528fe6f6b5.png)
Is there any way how to change the number of generated labels? For example instead of 3 values generate 6 or 10 values? 

I know that I can calculate my own labels, but I am trying to use as much inbuilt functionality as possible :)

Thanks,
Peter

**Comment by mgramley** [comment]
Any solution to this? Still struggling with it myself

**Comment by beto-rodriguez** [comment]
@mgramley See https://github.com/beto-rodriguez/LiveCharts2/issues/1178, does that help?
